hugr/Cargo.toml

```diff
@@ -22,7 +22,9 @@ bench = false
 path = "src/lib.rs"
 
 [features]
+default = ["proptest"]
```

Comment:
Is it possible to turn the proptest feature on only when `#[cfg(test)]` is true, maybe?

Reply:
Also pondering how well this agrees with what's in `justfile`